src/phoenix/trace/trace_dataset.py

```diff
@@ -15,13 +17,24 @@
 ]
 
 
+class ComputedColumns(Enum):
+    "The latency of the span in milliseconds"
+    latency_ms = "latency_ms"
+
+
 def normalize_dataframe(dataframe: DataFrame) -> "DataFrame":
     """Makes the dataframe have appropriate data types"""
 
     # Convert the start and end times to datetime
     dataframe["start_time"] = pd.to_datetime(dataframe["start_time"])
     dataframe["end_time"] = pd.to_datetime(dataframe["end_time"])
 
+    # Computed columns
+    dataframe[ComputedColumns.latency_ms.value] = (
```

Comment:
did you mean duration (as in `SpanColumn(Enum)`)?

Reply:
Oh thanks for the catch 😓